frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -96,6 +97,8 @@ public final class EnhancedGraphTable extends BaseEnhancedGraphTable
       DataProvider,
       Disposable {
 
+  private static final AtomicBoolean ENQUEUING_UPDATES_ENABLED = new AtomicBoolean(true);
```

Comment:
Hmmm theoretically it doesn't need to be static, it just needs to be a per-instance (i.e. per-project) lock, right? 🤔 (shouldn't make much difference in the actual use, but still)